tests/test.py

```diff
@@ -400,14 +400,16 @@ def test_int_long(self):
 
     def test_fileish_types(self):
         """Test the constructor does the right thing when given different types"""
-        with FitFile(testfile('Settings.FIT')):
+        with FitFile(testfile('Settings.fit')):
```

Comment:
This was my bad. The uppercase here was intentional to trigger [an issue](https://github.com/dtcooper/python-fitparse/issues/29#issuecomment-312436350) where `.FIT` in filename was confused with [`.FIT` in the file header](https://github.com/dtcooper/python-fitparse/blob/de90d9a4af700325f1681ba454a2fab39a3f6774/fitparse/base.py#L113). I thought I had added a `Settings.FIT` file alongside `Settings.fit`, but I guess it didn't get added to git properly (probably my filesystem's fault).

Would you be able to rename the file to `Settings.FIT` and change all the other instances of `Settings.fit` in the tests to `Settings.FIT`? Alternatively just add another file with an 8-character filename and an uppercase extension (`nametest.FIT`?) and sub that in here. Should add a comment too since I didn't really explain the test very well.